api/services/tasks/v1/tasks.proto

```diff
@@ -49,14 +49,10 @@ service Tasks {
 }
 
 message CreateTaskRequest {
-	// NOTE: reserve field 1 for task id.
-
-	// ContainerID specifies the container to use for creating this task.
-	//
 	// The spec from the provided container id will be used to create the
 	// task associated with this container. Only one task can be run at a time
 	// per container.
-	string container_id = 2;
+	string id = 1;
```

Comment:
missing a description of `id`

I'm assuming this is the id to be given to the task?

But if that's the case, how do you know which container to create that task from? So this must be the container id, in which case, I'll leave it named `container_id`